docs/data-access/data-access.md

```diff
@@ -19,9 +19,9 @@ As a prerequisite to applying for DACO access, please ensure that you:
 
 ## How to Apply for Access through DACO
 
-The instructions for applying for DACO can be found here on the [ICGC DACO website](https://icgc.org/daco), as well as a selection of [help guide videos](https://icgc.org/daco/help-guide-section).
+The instructions for applying for DACO can be found here on the [ICGC DACO website](https://daco.icgc.org/), as well as a selection of [help guide videos](https://daco.icgc.org/help-guides/daco-help-guides/).
 
-1. Register a DACO account at [www.icgc.org/user/register](https://icgc.org/user/register). If you already have an account, you can log in from here [www.icgc.org/user/login](https://icgc.org/user/login).
+1. Register a DACO account at https://daco.icgc.org/user/#/register. If you already have an account, you can log in from https://daco.icgc.org/user/#/login. 
 1. After registering, you will receive an email instructing you to set your Central User Directory password. Click the link from the email to set up your account password.
```

Comment:
Is this still the case? If so, we need to get them to change the "Central User Directory" thing - it's not clear in the ICGC context what that would be - it should be an ICGC account

Reply:
its still using the CUD, and they are still creating an icgc account.  the underlying tech hasnt changed.

Reply:
I know..just a shame they still reference "CUD" as that's really only an OICR thing - connects all the OICR products. That was my understanding anyway

Reply:
Iknow, but there is nothing i can do about this yet.  When we get a new daco contract, be it with austrailia or joseph, then we can change this.  right now, there is nothing we can do.